dcat/index.html

```diff
@@ -465,6 +470,7 @@ <h3>Basic example</h3>
   a dcat:Distribution&nbsp;;
   dcat:downloadURL &lt;http://www.example.org/files/001.csv&gt; ;
   dcterms:title "CSV distribution of imaginary dataset 001"@en ;
+  dcterms:title "distribución 001, en CSV, del conjunto de datos imaginario"@en ;
```

Comment:
```suggestion
  dcterms:title "distribución, en CSV, del conjunto de datos imaginario 001"@en ;
```

Reply:
@agbeltran , I wonder whether the distribution title should be revised as above.

Reply:
thanks, agreed!

Reply:
@andrea-perego done!

Reply:
Thanks!